core/src/main/scala/spinoco/fs2/cassandra/internal/ctype/BigIntCType.scala

```diff
@@ -0,0 +1,43 @@
+package spinoco.fs2.cassandra.internal.ctype
+
+import com.datastax.oss.driver.api.core.ProtocolVersion
+import com.datastax.oss.driver.api.core.`type`.DataType
+import com.datastax.oss.driver.internal.core.`type`.codec.BigIntCodec
+import scodec.bits.BitVector
+import scodec.{Attempt, Codec, DecodeResult, SizeBound}
+import spinoco.fs2.cassandra.{CType, util}
+
+object BigIntCType {
+
+  def instance(cqlDataType: DataType): CType[Long] = {
+    val nativeCodec = new BigIntCodec()
+    new CType[Long] {
+      def cqlType: DataType = cqlDataType
+
+      def cqlCodec(protocolVersion: ProtocolVersion): Codec[Long] = {
```

Comment:
Can you just add clarification comment sommewhere that naming follws casandra bigint type that is in fact Java's Long ?

Reply:
Like this?:
```
/**
* This naming follows Cassandra BigInt semantics.
* This corresponds to Scala Long.
*/
object BigIntCType {
```